keylime-agent/src/main.rs

```diff
@@ -691,23 +690,24 @@ async fn main() -> Result<()> {
             )));
         }
 
-        let keylime_ca_cert = match crypto::load_x509(&ca_cert_path) {
-            Ok(t) => Ok(t),
-            Err(e) => {
-                error!(
-                    "Failed to load trusted CA certificate {}: {}",
-                    ca_cert_path.display(),
-                    e
-                );
-                Err(e)
-            }
-        }?;
+        let keylime_ca_certs =
+            match crypto::load_x509_cert_chain(&ca_cert_path) {
+                Ok(t) => Ok(t),
+                Err(e) => {
+                    error!(
+                        "Failed to load trusted CA certificate {}: {}",
+                        ca_cert_path.display(),
+                        e
```

Comment:
clippy should complain here too, shouldn't?

Reply:
No because you cannot call functions currently in a fromat string, I think.

Reply:
Right. But I mean "e"

Reply:
Yeah, I don't know. I would keep it this way for now, because mixing the two styles looks wrong.